crates/red_knot_python_semantic/resources/mdtest/union_types.md

```diff
@@ -121,7 +146,7 @@ Simplifications still apply when `Unknown` is present.
 from knot_extensions import Unknown
 
 def _(u1: str | Unknown | int | object):
-    reveal_type(u1)  # revealed: Unknown | object
+    reveal_type(u1)  # revealed: object
```

Comment:
The new simplified result here kind of defeats the purpose of the original test, if I correctly understand its purpose. Should this be changed to something like `int | Unknown | bool` -> `Unknown | int`?

Reply:
Yes, using `int | Unknown | bool` -> `Unknown | int` looks right to me.